delivery_slip_packages/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,5 @@
+
+> - [Sygel](https://www.sygel.es):
+>
+> > - Juan Alberto Raja Martinez\<<juan.raja@sygel.es>\>
+> > - Valentín Vinagre Urteaga\<<valentin.vinagre@sygel.es>\>
```

Comment:
@manuelregidor don't you have to be? Do emails have to be?

Reply:
As far as I know, we don't have a contributors format, so I think that writing the email addresses is totally fine. I don't need to be in the contributors list as I just made a review of the module.

Reply:
@BetoRaja Delete "personal" emails. We should be contacted at the company level. Just like OCA.

Reply:
@ValentinVinagre Personal email addresses are used in OCA too. There are plenty of example (i.e. https://github.com/OCA/l10n-spain/blob/17.0/l10n_es_aeat/readme/CONTRIBUTORS.md). But OK, I'll change it, as @BetoRaja has finished already for today and this module needs to be deployed.